CONTRIBUTING.md

```diff
@@ -1,256 +1,339 @@
-- [3. Teamwork](#3-teamwork)
-    - [3.1 Update CODEOWNERS](#31-update-codeowners)
-    - [3.2 Single package per pull request](#32-single-package-per-pull-request)
-    - [3.3 Open issues and pull request expiration](#33-open-issues-and-pull-request-expiration)
-    - [3.4 Understanding labels](#34-understanding-labels)
+- [Etiquete](#etiquete)
+- [Opening issues and discussions](#opening-issues-and-discussions)
+  - [Reporting a bug](#reporting-a-bug)
+  - [Reporting an outdated package](#reporting-an-outdated-package)
+  - [Requesting enhancements or new functionality](#requesting-enhancements-or-new-functionality)
+  - [Issue closing](#issue-closing)
+- [Opening pull requests](#opening-pull-requests)
+  - [Pull request title](#pull-request-title)
+  - [Pull request body](#pull-request-body)
+- [Conventions / Guidelines](#conventions--guidelines)
+  - [Manual or automatic](#manual-or-automatic)
+  - [Package types](#package-types)
+  - [Embedded packages](#embedded-packages)
+  - [Semi-Embedded packages](#semi-embedded-packages)
+  - [Remote packages](#remote-packages)
+  - [Metadata](#metadata)
+    - [Naming](#naming)
+    - [Dependency versions](#dependency-versions)
+    - [Conform to guidelines](#conform-to-guidelines)
+    - [Obligatory metadata](#obligatory-metadata)
+    - [Obligatory tags and categories](#obligatory-tags-and-categories)
+    - [Description](#description)
+    - [Maintainers](#maintainers)
+    - [Icons](#icons)
+    - [Chocolatey compatibility](#chocolatey-compatibility)
+  - [Chocolatey Scripts](#chocolatey-scripts)
+    - [UI Automation](#ui-automation)
+      - [Work on all locales](#work-on-all-locales)
+      - [Avoid brittle scripts](#avoid-brittle-scripts)
+    - [Source Files](#source-files)
+      - [Encoding](#encoding)
+      - [Code style](#code-style)
+  - [Update Script](#update-script)
+    - [Use `UseBasicParsing`](#use-usebasicparsing)
+    - [Do not download large files](#do-not-download-large-files)
+- [FAQ](#faq)
+  - [Why do we automatically close issues?](#why-do-we-automatically-close-issues)
+  - [Why do not the repository maintainers fix the issues?](#why-do-not-the-repository-maintainers-fix-the-issues)
+  - [How can I create a fix version of a package?](#how-can-i-create-a-fix-version-of-a-package)
 
 </details>
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 <!-- markdownlint-enable -->
 
-# 1. Packages
+## Etiquete
 
-## 1.1 Basics
+Make sure that all communications you make on this repository follows our [CODE OF CONDUCT](https://github.com/chocolatey-community/.github/blob/main/CODE_OF_CONDUCT.md). Failure to adhere to this document may lead you to be banned from the repository.
+In addition to the code of conduct document, make sure that you do not make unneeded reference to other users when opening issues, discussions or pull requests, doing so may lead to the issue or pull request being ignored by these users instead.
```

Comment:
It would be good to explicitly state that references means @-ing a user.

Reply:
Its proper term is to  [mention](https://github.blog/2011-03-23-mention-somebody-they-re-notified/).